src/functionBased/components/TodoItem.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FaTrash } from 'react-icons/fa';
+import styles from './TodoItem.module.css';
+
+const TodoItem = ({ ...props }) => {
+  const [editing, setEditing] = useState(false);
+
+  const handleEditing = () => {
+    setEditing(true);
+  };
```

Comment:
i think since this function contains only a call to another function you can call the function directly like this:
`onDoubleClick={setEditing(true);}`

Reply:
Thanks for noticing this one.